### PR TITLE
Changes on components to solve issues

### PR DESCRIPTION
**Pull request by Craaftx**
Link to issue : https://github.com/Opla/front/issues/67 https://github.com/Opla/front/issues/94 https://github.com/Opla/front/issues/93

- Change display to be more in line with material design ✔️
- Use _main.css_ for component instead of file constants ✔️
- Change css rules for _fileInput_ class ✔️
- (Optionnal) Use image color to create a background color ✔️

### CODE REVIEW

**Comment by Craaftx** [comment]
FileInput has is own margin who cause problem when you want center him vertically and horizontally
bugfr
It's better to not declare margin in the fileInput class but rather locally when uses it.

**Comment by Craaftx** [comment]
I replaced FileInput's css class, so we do not need to make changes to it and it does not affect it.

**Comment by Craaftx** [comment]
Done, ready for review @mikbry

File: src/public/css/main.css [hunk 1]
```diff
@@ -687,19 +687,19 @@
  }
  
  /* templatesList (tpl) Component Style */
+.tpl-templateBox {
```
Comment (mikbry) [inline src/public/css/main.css]: naming should be .opla_template-box instead same fornat for other classes names

File: src/public/css/main.css [hunk 2]
```diff
@@ -687,19 +687,19 @@
...
+  height: 250px;
+  outline: 1px solid transparent;
+  text-decoration: "none";
+}
+
  .tpl-infoC {
```
Comment (mikbry) [inline src/public/css/main.css]: .opla_template-infoC

File: src/public/css/main.css [hunk 3]
```diff
@@ -687,19 +687,19 @@
...
   height: 180px;
 }
 
-.tpl-templateBox {
-  height: 250px;
-  outline: 1px solid transparent;
-  text-decoration: "none";
-}
-
 .tpl-anchor {
```
Comment (mikbry) [inline src/public/css/main.css]: .opla_template-anchor

**Comment by Craaftx** [comment]
I change that and add some little issue to this PR

**Comment by Craaftx** [comment]
Damn it, change that ..

**Pull request merged**